simuq/providers.py

```diff
@@ -55,17 +55,15 @@ def compile(self, qs, provider, device, aais, tol=0.01, trotter_num=6, verbose=0
         if self.provider == "quera":
             nsite = qs.num_sites
 
-            if aais == "rydberg1d_global":
-                from simuq.aais.rydberg1d_global import GenMach
+            if aais == 'rydberg1d_global' :
```

Comment:
Why the switch to single quotes? We've conventionally been using double quotes.

Reply:
Pycharm's formatter did it, lol. I can revert for uniformity for now